package.json

```diff
@@ -989,7 +990,10 @@
           "mod-settings.entries.item.post",
           "mod-settings.entries.item.put",
           "mod-settings.global.read.ui-inventory.display-settings.manage",
-          "mod-settings.global.write.ui-inventory.display-settings.manage"
+          "mod-settings.global.write.ui-inventory.display-settings.manage",
+          "audit.config.groups.collection.get",
+          "audit.config.groups.settings.audit.inventory.collection.get",
+          "audit.config.groups.settings.item.put"
```

Comment:
The permission `audit.config.groups.settings.audit.inventory.item.put` is also needed.